pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentTopic.java

```diff
@@ -829,11 +829,14 @@ public boolean isActive() {
     }
 
     @Override
-    public void checkGC(int maxInactiveDurationInSec, InactiveTopicDeleteMode deleteMode) {
-        if (!deleteWhileInactive) {
+    public void checkGC(int maxInactiveDurationSeconds, InactiveTopicDeleteMode inactiveTopicDeleteMode) {
+        if (!isDeleteWhileInactive()) {
```

Comment:
If we already use InactiveTopicPolicies, I think we don't need to pass `maxInactiveDurationSeconds` and `inactiveTopicDeleteMode` in the param list. Of course, if only use InactiveTopicPolicies to maintain the namespace level policy, we still need to pass parameters.